### PR TITLE
DOC-2283: replace RedisTimeSeries with Time Series

### PR DESCRIPTION
**Pull request by dwdougherty**
This commit is for the module de-branding effort.

### CODE REVIEW

File: docs/docs/_index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ type: docs
 [![Discord](https://img.shields.io/discord/697882427875393627?style=flat-square)](https://discord.gg/KExRgMb)
 [![Github](https://img.shields.io/static/v1?label=&message=repository&color=5961FF&logo=github)](https://github.com/RedisTimeSeries/RedisTimeSeries/)
 
-RedisTimeSeries is a Redis module that adds a time series data structure to Redis.
+Time Series is a Redis module that adds a time series data structure to Redis.
```
Comment (LiorKogan) [inline docs/docs/_index.md:11]: We cannot say that Time Series is a module.

File: docs/docs/_index.md [hunk 2]
```diff
@@ -26,10 +26,10 @@ See the [clients page](clients) for the full list.
 
 ## Using with other metrics tools
 
-In the [RedisTimeSeries](https://github.com/RedisTimeSeries) organization you can
-find projects that help you integrate RedisTimeSeries with other tools, including:
+In the [Time Series](https://github.com/RedisTimeSeries) organization you can
```
Comment (LiorKogan) [inline docs/docs/_index.md:29]: This is a name of a GitHub organization. We are not changing it for now.

Comment (dwdougherty) [reply docs/docs/_index.md:29]: I've fixed this in my local branch and I'll update the PR once a few more comments come in.

File: docs/docs/_index.md (hunk 1 above)
Comment (elena-kolevska) [reply docs/docs/_index.md:11]: @dwdougherty we'll probably have to do the same change to all the other features.

**Comment by dwdougherty** [comment]
Changes were covered by other commits.

**Pull request closed without merging**